week-2-wide-perspective/w02e05-simple-server-side/src/error.handler.ts

```diff
@@ -0,0 +1,25 @@
+import { logger } from './logger.js'
+import { Request, Response, NextFunction } from 'express'
+import { HTTPError, ValidationError } from './classes/Error.js'
+
+export const errorHandler = (err: Error, req: Request, res: Response, next: NextFunction) => {
+    const { message: errorMessage, stack: errorStack = '' } = err
+    let status = 500
+    let errors = undefined
+    let message = errorMessage
+    let stack = process.env.NODE_ENV === 'development' ? errorStack.split('\n') : undefined
```

Comment:
to chyba może być `const` - ale generalnie fajnie że dołączasz tylko `stack` w momencie `"development"` 👌